bazel/asn1c.bzl

```diff
@@ -50,13 +50,17 @@ def _construct_asn1c_commands(ctx, dir_path):
     """Return a string of command that runs asn1c"""
     flags = ctx.attr.flags
     asn1_file = ctx.attr.asn1_file.files.to_list()[0].path
-    asn1c_command_template = "{asn1c} {flags} -D {dir} {input}"
+
+    # TODO: GH13021 this is a mitigation for excessive logging that should be handled more clean
```

Comment:
It might make sense not to let this filter trickle into the build definitions, but wrap the respective `bazel build` and `make` invocations.